tests/lib/snaps/test-snapd-joystick/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-snapd-joystick
+version: 1.0
+summary: Basic joystick snap
+description: A basic snap which allows reading and writing to joystick devices
+
+apps:
+  sh:
+    command: ../../../bin/sh
```

Comment:
While I personally like those myself, thinking of the command as a skilful hack, Gustavo has requested that we don't do this anymore (that is, use ../../../ to get to /bin/sh). 

CC @chipaca 

What is the right way to refer to /bin/sh from a snap's command given that it seems to be relative to $SNAP.

Reply:
we've not done the work to drop ../../../bin/sh and move to /bin/sh

Reply:
Thanks @chipaca